src/manage_sql.c

```diff
@@ -54400,6 +54421,14 @@ update_from_slave (task_t task, entity_t get_report, entity_t *report,
                                     entity_text (host_start));
         }
       hosts = next_entities (hosts);
+
+      current_commit_size ++;
```

Comment:
Don't think there should be a space.

Reply:
The extra space has been removed in fd2fb81.